### PR TITLE
N51: RustNL

### PR DESCRIPTION
**Pull request by Vrixyz**
_Part of #1505_

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
The main relevant news was the discussion of cross-project collaboration on foundational GUI building blocks and OS abstractions. Linebender and Robius are the key gathering points for that initiative.

**Comment by Vrixyz** [comment]
@alice-i-cecile I assume that's the talk you're mentioning (Xilem: let's build high performance Rust UI) : https://www.youtube.com/watch?v=521NfGf7AR0&t=19308s

**Comment by tertsdiepraam** [comment]
There was also the bit where Michael Melchiore from Airbus mentioned that they use Bevy's ECS, which was super cool.

@Vrixyz yes that talk and the talk by Kevin Boos about Robius.

**Comment by tertsdiepraam** [comment]
We'll also put the talks online separately in the near future of course.

File: content/news/051/index.md [hunk 1]
```diff
@@ -85,6 +85,27 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+While the conference was not specific to game development,
+organizers contacted rust game development communities
+to showcase games developed in Rust between talks,
+which you can [watch on youtube][rustnl_games_youtube].
+
+Special mention to the following talks, relevant to game development:
+- [The talk from Kevin Boos][rustnl_kevin_boos_talk]: let's build high performance Rust UI
+- [The talk from Ralph Levien][rustnl_xilem_talk] about [Xilem]
+[rustnl_xilem_intro]: a research project on Rust user interface architecture.
```
Comment (alice-i-cecile) [inline content/news/051/index.md:100]: This seems malformed: should this be another bullet point?

Comment (Vrixyz) [reply content/news/051/index.md:100]: Ah thanks! it's a misplaced semantic line break

**Comment by janhohenheim** [comment]
Ping me when this is ready for merging :)

**Comment by Vrixyz** [comment]
@janhohenheim I think it's ready, I didn't add a flavor image, do you think it's worth it ? Not sure what to include though.

We could also wait for the individual talks to be uploaded (or make a followup PR)

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**